### PR TITLE
Fix Windows shortcut

### PR DESCRIPTION
**Pull request by levirs565**
This fix Windows Shortcut problem. To create shortcut run this code in python:
```python
from guiscrcpy.platform.windows_tools.tools import make_shortcut
make_shortcut()
```

Changes:
- Minimum Python version is Python 3.7
- `guiscrcpy_logo_SRj_icon.ico` is added to package_data
- Add `guiscrcpy-noconsole` gui scripts
- Use wrapped Popen function to run command

### CODE REVIEW

**Review by srevinsaju — COMMENTED** [review]
Interesting. Thanks for the Pull Request. Looks good, but a few doubts

File: guiscrcpy/lib/utils.py [hunk 1]
```diff
@@ -162,3 +163,14 @@ def show_message_box(text, info_text="", buttons=QMessageBox.Ok):
...
+def open_process(command, stdin=None, stdout=None, stderr=None, cwd=None):
+    # warning: CREATE_NO_WINDOW is added in Python 3.7
+    return Popen(
+        command,
+        stdin=stdin,
+        stdout=stdout,
+        stderr=stderr,
+        cwd=cwd,
+        creationflags=CREATE_NO_WINDOW
+    )
```
[the remark this answers is not in the document]
Comment (levirs565) [reply guiscrcpy/lib/utils.py:176]: To prevent console window (cmd.exe) appear

**Comment by levirs565** [comment]
I haven't checked whether all the features work well.

File: guiscrcpy/lib/utils.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (srevinsaju) [reply guiscrcpy/lib/utils.py:176]: Ok, looks good, but I would like to make some more changes:
```python
def open_process(command, stdin=None, stdout=None, stderr=None, cwd=None):
    # warning: CREATE_NO_WINDOW is added in Python 3.7
    return Popen(
        command,
        stdin=stdin,
        stdout=stdout,
        stderr=stderr,
        cwd=cwd,
        creationflags=CREATE_NO_WINDOW
    )
```
can be better
```python
def open_process(*args, **kwargs):
    # warning: CREATE_NO_WINDOW is added in Python 3.7
    if ( is_windows ) and (python version greater than 3.7) :
         return Popen(*args, **kwargs, creationflags=CREATE_NO_WINDOW)
    else:
         return Popen(*args, **kwargs)
    
```

**Comment by srevinsaju** [comment]
> I haven't checked whether all the features work well.

The tests would ensure that all features work properly, right now.. the tests have failed :P 
looks like a minor formatting problem. Should be solved by running `black`.

```bash
black guiscrcpy
```
would format all of them.

File: guiscrcpy/lib/utils.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (srevinsaju) [reply guiscrcpy/lib/utils.py:176]: I have given only logical cues, I am sure you will make out how to add those too :smile: .

[the remark this answers is not in the document]
Comment (srevinsaju) [reply guiscrcpy/lib/utils.py:176]: And additionally, you can remove the comment `# warning: ....` once you have added that fix.

**Comment by levirs565** [comment]
I have finished improving the code. But there are Codacy security issues

**Comment by srevinsaju** [comment]
@levirs565 the tests are still failing: https://github.com/srevinsaju/guiscrcpy/pull/205/checks?check_run_id=1681436824

**Comment by levirs565** [comment]
Please check latest commit tests. This has been fixed. See this test https://github.com/srevinsaju/guiscrcpy/pull/205/checks?check_run_id=1681483802

**Comment by srevinsaju** [comment]
thanks, looks good. I will do a final review and merge.

**Review by srevinsaju — APPROVED** [review]

**Pull request merged**